read_mars/__init__.py

```diff
@@ -42,3 +42,35 @@ def read_mars(filename, tree='Events', verbose=False):
     f.Close()
 
     return pd.DataFrame(events)
+
+
+def read_mars_fast(filename, tree='Events', leaves=[]):
+    """Return a Pandas DataFrame of a star or ganymed output root file.
+    
+    A faster (~factor 15) version of read_mars. It also omits the useless leaves fBits and fUniqueID. 
+    Keyword arguments:
+    tree -- Set, which tree to read. (Default = "Events")
+    leaves -- Specify a list of leaves. (Default is [], what reads in all leaves)
+    """
+
+    f = ROOT.TFile(filename)
+    tree = f.Get(tree)
+    if not leaves:
+        leaves = [l.GetName() for l in tree.GetListOfLeaves() if
+            not (l.GetName().endswith('.') or l.GetName().endswith('fBits') or l.GetName().endswith('fUniqueID'))]
```

Comment:
I don't think they have any meaning in the pandas DataFrame, because they show the same values for all branches. As they are automatically interpreted as double, they do also block a considerable amount of memory. (almost 1 third)
Maybe we need further confirmation, that these leaves are in fact useless in a pandas DataFrame.